src/action/services/useLocalStorage.js

```diff
@@ -0,0 +1,37 @@
+function useLocalStorage() {
+  const saveToLocalStorage = (key, value) => {
+    const convertedValue = Array.isArray(value)
+      ? value
```

Comment:
WTF IS EVEN THAT!?
Change it to something more readable. THIS IS WRONG IN SO MANY LEVELS TO CREATE A TRINERY WITHIN A TRINERY!!!!!!!~!

Reply:
its suppose to be a fun interactive riddle for the code reviewr.
anyway modified it to be a switch case statement